code/modules/paperwork/photocopier.dm

```diff
@@ -227,6 +279,41 @@
 	p.pixel_x = rand(-9, 9)
 	return p
 
+
+/obj/machinery/photocopier/MouseDrop_T(mob/target, mob/user)
+	check_ass() //Just to make sure that you can re-drag somebody onto it after they moved off.
+	if (!istype(target) || target.buckled || get_dist(user, src) > 1 || get_dist(user, target) > 1 || user.stat || istype(user, /mob/living/silicon/ai) || target == ass)
+		return
+	src.add_fingerprint(user)
+	if(target == user && !user.stat && !user.weakened && !user.stunned && !user.paralysis)
```

Comment:
Instead of `!user.stat && !user.weakened && !user.stunned && !user.paralysis` this should check `!user.incapacitated()`